x-pack/plugins/fleet/server/saved_objects/index.ts

```diff
@@ -460,6 +460,10 @@ const getSavedObjectTypes = (): { [key: string]: SavedObjectsType } => ({
           dynamic: false,
           properties: {},
         },
+        latest_install_failed_attempts: {
+          dynamic: false,
+          properties: {},
+        },
```

Comment:
Mapping additions require to add an associated model version. Please refer to [the modelVersion documentation](https://github.com/elastic/kibana/blob/9b59e84d3e3601a12fa5ff33814393b8f7644010/packages/core/saved-objects/core-saved-objects-server/docs/model_versions.md) for the guidelines and examples

Reply:
Thanks @pgayvallet, I had a second though on this one and there is no need for us to index that field as we have a default value there is nothing to do here correct?

Reply:
I added the model version with the mapping addition, I think the doc is clear was my first time adding a mapping since this was introduced, trying to figure this out

Reply:
Okay 🙂. FWIW, any feedback on the doc is always good to take, so I was genuinely asking.